src/compiler/checker.ts

```diff
@@ -9703,12 +9703,14 @@ namespace ts {
 
                 // fill in any as-yet-unresolved late-bound members.
                 const lateSymbols = createSymbolTable() as UnderscoreEscapedMap<TransientSymbol>;
-                for (const decl of symbol.declarations) {
-                    const members = getMembersOfDeclaration(decl);
-                    if (members) {
-                        for (const member of members) {
-                            if (isStatic === hasStaticModifier(member) && hasLateBindableName(member)) {
-                                lateBindMember(symbol, earlySymbols, lateSymbols, member);
+                if (symbol.declarations) {
+                    for (const decl of symbol.declarations) {

```

Comment:
```suggestion
                    for (const decl of symbol.declarations || emptyArray) {

```
Seems a smidge cleaner; so long as it doesn't trigger any unreachable branch stuff.

Reply:
Doesn't seem to. Done.